pkg/minikube/registry/drvs/podman/podman.go

```diff
@@ -167,3 +174,8 @@ func status() registry.State {
 
 	return registry.State{Error: err, Installed: true, Healthy: false, Doc: docURL}
 }
+
+func sudoNeedsPassword() bool {
+	err := exec.Command("sudo", "-n", "ls").Run()
```

Comment:
is there a cheaper way than executing a command ? (since this code is used in driver check it would worth to optimize second to miliseconds

how about this one ? https://askubuntu.com/questions/357220/how-to-check-if-sudo-password-has-been-entered-for-this-terminal-session#:~:text=If%20a%20password%20is%20required,2%3E%2Fdev%2Fnull%20.
```
if sudo -n true 2>/dev/null```